lib/patch-series.ts

```diff
@@ -117,6 +117,8 @@ export class PatchSeries {
 
     public static async getFromNotes(notes: GitNotes,
                                      pullRequestURL: string,
+                                     pullRequestTitle: string,
+                                     pullRequestSubmitter: string,
                                      pullRequestDescription: string,
```

Comment:
If you like.

In my understanding, PR consists of title _and_ description.

Reply:
> In my understanding, PR consists of title _and_ description.

That obviously disagrees with the usage of the term "PR description" in current `master`.

Reply:
History does show `const description = `${pr.title}\n\n${pr.body}`;` in ci-helper.ts.  Moving from strings to objects is a good thing.  

One of the commits indicates a `\n\n` in the pr.body could cause a problem.  The explanation was not clear.  Since the title was always pre-pended with a pair of lf, was this a regex greedy problem?

Reply:
Yes, regex is greedy, which is a good thing according to code's intention: it should split on the very last `\n\n` to separate footer from the rest